pep-0605.rst

```diff
@@ -0,0 +1,875 @@
+PEP: 605
+Title: A rolling feature release stream for CPython
+Version: $Revision$
+Last-Modified: $Date$
+Author: Steve Dower <steve.dower@python.org>, Nick Coghlan <ncoghlan@gmail.com>
+Discussions-To: Link TBD
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 20-Sep-2019
+Python-Version: 3.9
+Post-History: 30-Sep-2019
+
+
+Abstract
+========
+
+Rather than proposing a change to CPython's release cadence (as PEP 602 does),
+or a policy change to allow backwards compatible feature additions later in a
+release series (as PEP 598 does), this PEP instead proposes that we create a
+rolling stream of production-ready beta releases by amending CPython's
+pre-release management process as follows:
+
+* Feature freeze, ABI freeze, pyc file format freeze, and maintenance branch
+  creation all correspond with the first release candidate for an X.Y.0 release
+* The X.Y.0 release candidate period is extended from 3 weeks to 2 months
+* Beta releases from the master git branch would occur every 2 months
+  whenever there is no X.Y.0 release in the release candidate phase
+* Routine alpha releases are removed from the process
+* Release managers may still choose to mark particular releases as alpha
+  releases if they feel such releases are appropriate. If any such alpha
+  releases are made, it is expected that they would be in place of one of the
+  scheduled beta releases
+* The deprecation policy for interfaces that have been included in an X.Y.0
```

Comment:
PEP 387 should handle the deprecation policy specification.

Reply:
Done